packages/liveblocks-core/src/connection.ts

```diff
@@ -306,9 +306,13 @@ function logPrematureErrorOrCloseEvent(e: IWebSocketEvent | Error) {
 }
 
 function logCloseEvent(event: IWebSocketCloseEvent) {
+  const details = [`code: ${event.code}`];
```

Comment:
did you mean to have this change as part of this PR?

Reply:
I wanted to include this small fix in 1.11.2 too, yes. Should have made that more obvious.